quota/quota.go

```diff
@@ -33,3 +33,19 @@ func IsUserWithinQuota(ctx rcontext.RequestContext, userId string) (bool, error)
 
 	return true, nil // no rules == no quota
 }
+
+func GetUserUploadMaxSizeBytes(ctx rcontext.RequestContext, userId string) int64 {
```

Comment:
This could probably easily go in `upload/upload.go`, but got lazy. Shout if this function needs moving.